app/src/main/java/org/lineageos/selfie/SharedPreferencesExt.kt

```diff
@@ -271,6 +271,19 @@ internal var SharedPreferences.aspectRatio: Int
         }
     }
 
+// Custom storage location
+private const val CUSTOM_STORAGE_LOCATION = "custom_storage_location"
+
+internal var SharedPreferences.customStorageLocation: Uri?
+    get() {
+        return runCatching { Uri.parse(getString(CUSTOM_STORAGE_LOCATION, null)) }.getOrNull()
```

Comment:
If we ever store the Uri it'll be valid, or null
No need for catching, get the raw pref, if null/empty early return, otherwise parse it

Reply:
Fix'd.